www/src/pages/guidelines/typography.js

```diff
@@ -0,0 +1,600 @@
+      <Columns>
+        <CopyColumn sticky={false}>
+          <p>
+            Gatsby uses the all-time classic Futura for headlines and display
+            copy. Long form text such as articles and documentation currently
+            use the native font stack, but we are actively looking for a serif
+            to increase the reading experience.
+          </p>
+        </CopyColumn>
+        <ContentColumn css={{ alignSelf: `flex-end`, display: `flex` }}>
+          <Flex flexDirection="column">
+            <Typeface fontFamily="header">Futura PT</Typeface>
+          </Flex>
+        </ContentColumn>
+      </Columns>
+      <Columns>
+        <CopyColumn>
+          <SectionSubheading>The native font stack</SectionSubheading>
+          <p>
+            The “native font stack” depends on the user’s operating system and
+            device; depending on that, we use San Francisco UI, Roboto or Segoe
+            UI.
+          </p>
+          <Text as="p">
+            Our monospace font stack also makes use of the default fonts
+            available: That’s usually San Francisco Mono, Menlo or Monaco on Mac
+            OS X, Consolas on Windows, or Liberation Sans on Linux
+            distributions.
+          </Text>
+          <Text as="p" mb={0}>
+            In our (Figma, Sketch, etc.) designs we use Roboto as it’s easily
+            available.
+          </Text>
+        </CopyColumn>
+        <ContentColumn css={{ alignSelf: `flex-end`, display: `flex` }}>
+          <Flex flexDirection="column">
+            <Typeface fontFamily="system">Sans-serif</Typeface>
+            <Typeface fontFamily="monospace">Monospace</Typeface>
+          </Flex>
+        </ContentColumn>
+      </Columns>
+    </Section>
+    <Section bg="yellow.40" pr={{ xxs: 0, xs: 0, sm: 0, md: 0, lg: 0 }}>
+      <SectionHeading>Scale</SectionHeading>
+      <Columns>
+        <CopyColumn pr={{ xxs: 5, xs: 6, sm: 7, lg: 0 }}>
+          <p>
+            Since our primary use for the design system is interface design, our
+            typographic scale is hand-crafted. This way we don’t have to worry
+            about decimals in our design tools or when calculating line heights,
+            don’t have to worry about subpixel rounding errors, and have total
+            control over which sizes exist instead of outsourcing that job to a
+            mathematical formula.
+          </p>
+          <p>
+            For long form content like articles, using a modular scale is
+            totally fine though. Typography.js makes setting up and using such a
+            scale easy.
+          </p>
+        </CopyColumn>
+        <ContentColumn fullWidth>
+          {theme.fontSizes.map((size, index) => (
+            <div key={`${index}-${size}`}>
+              <Text
+                // don't scale based on root font size
+                fontSize={`${parseFloat(size) * 16}px`}
+                color="grey.90"
```

Comment:
The grey on yellow colors + small text combination is pretty difficult to read and doesn't meet contrast ratio guidelines.

Reply:
Removed the yellow background!